### DISCUSSION TITLE
io.jsonwebtoken.impl.crypto.DefaultJwtSignatureValidator no longer available. Updating version from 0.11.5 -> 0.12.3

### DISCUSSION THREAD

**Discussion by senecal-jjs** [jwtk/jjwt#868] [Q&A] [answered]
What should replace the `DefaultJwtSignatureValidator` in the code below?
```
val secret = SecretKeySpec(secret.toByteArray(), SignatureAlgorithm.HS256.jcaName)
val validator = DefaultJwtSignatureValidator(SignatureAlgorithm.HS256, secret, Decoders.BASE64URL)
return validator.isValid(token, signature)
```

**Comment by lhazlewood** [comment, marked as answer]
Hi @senecal-jjs !

FWIW, this is why we're explicit in documentation that the `impl` `.jar` should never be in `compile` scope - there is no guaranteed API support in the `impl` module and JJWT users should never depend on it directly. (From the `0.11.5` documentation: https://github.com/jwtk/jjwt/tree/0.11.5#understanding-jjwt-dependencies` ).

So I'm not exactly sure what should be suggested vs what is already documented via https://github.com/jwtk/jjwt/tree/0.12.3#jws-read

If there is no exception, the token is valid. 😄
